ansible/roles/kube_nfs_volumes/README.md

```diff
@@ -0,0 +1,114 @@
+# kube_nfs_volumes
```

Comment:
This should go into openshift-ansible-contrib, it is a community contributed role.